presto-main/pom.xml

```diff
@@ -369,7 +369,7 @@
         <dependency>
             <groupId>io.jsonwebtoken</groupId>
             <artifactId>jjwt-jackson</artifactId>
-            <scope>runtime</scope>
+            <scope>compile</scope>
```

Comment:
you should be able to remove this line.  compile is the default scope.

Reply:
done